subscription-manager.spec

```diff
@@ -1112,6 +1105,13 @@ find %{buildroot} -name \*.py* -exec touch -r %{SOURCE0} '{}' \;
 %{python_sitearch}/rhsmlib/services/__pycache__
 %endif
 
+# syspurpose
+%dir %{python_sitearch}/syspurpose
```

Comment:
What is the purpose of this section? Not saying it is wrong, just want to know why it is needed.
I see that most of the lines around it were removed below, but these were retained.

Reply:
Good question. Some files were moved from `syspurpose/src/syspurpose` to the `src/syspurpose`. So these files are installed to the system (`/usr/lib64/python3.6/`) like `rhsmlib` is installed to the system. So the purpose of this section is to mark these installed files as part of subscription-manager RPM.